eng/SourceBuild.Version.Details.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Dependencies>
+  <ProductDependencies>
+    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="6.0.100-preview.6.21321.1" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
How will these versions get updated?

Reply:
This is temporary for our Preview6 build.  We will continue adding all repos to this file manually and update it from Preview6 manifests in the Preview6 timeframe.  Once the source-build dependency graph is incorporated into all repos, we'll remove this file and use `Version.Details.xml` directly to get the repos to build.